build-system/common/esbuild-babel.js

```diff
@@ -127,7 +127,7 @@ function getEsbuildBabelPlugin(
         const map = outputFiles.find(({path}) => path.endsWith('.map'));
 
         if (!map) {
-          debug('post-esbuild', code.path, code.text);
+          debug('post-esbuild', code?.path, code?.text);
```

Comment:
Unrelated changes?

Reply:
This got merged due to auto-merge being enabled.
This is an unrelated change but I have seen situations where code is `undefined` due to some build error that this then masks. Due to that I would rather leave this in place then revert but I'm fine with that as well.